starboard/nplb/posix_compliance/posix_file_close_test.cc

```diff
@@ -23,8 +23,17 @@ namespace nplb {
 namespace {
 
 TEST(PosixFileCloseTest, CloseInvalidFails) {
-  int fd = -1;
-  EXPECT_FALSE(close(fd) == 0);
+  const int fd = -1;
```

Comment:
nit: `constexpr int kInvalidFD`= -1;`

This is so that the later close can read even more evident: `close(kInvalidFD);`